src/parser/mod.rs

```diff
@@ -4541,6 +4561,18 @@ impl<'a> Parser<'a> {
             return Ok(vec![]);
         }
 
+        if end_token == Token::SemiColon
+            && self
```

Comment:
do we also have to check the parser options here too?

Reply:
It might be that we should _only_ be checking the parser options. I'll investigate

Reply:
Switched to parser options 👍